package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-billing",
-  "version": "2.4.0",
+  "version": "2.4.2",
```

Comment:
I'll review this PR in more detail later. But please do not manually update the version number in PR.

Reply:
Why is this? As part of these changes we expect the version to be incremented. It doesn't make sense for us to downgrade this as we increment it for proper cache/references in lockfiles.

Reply:
What you do for forks is not up to me. But for upstream, i.e. this repository, I want to increment the version myself. This is because I might add other changes along with this version.

Reply:
Makes sense! Thanks @cbrevik -- we will update this.